src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const spouseSex = gender => (gender === 'm' ? 'wife' : 'husband');
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${spouseSex(sex)}`
+          : 'I am not married'}
```

Comment:
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
      
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
      
      <p className="Person__partner">
        {isMarried
          ? `${partnerName} is my ${spouseSex(sex)}`
          : 'I am not married'
        }
```
Fix formatting to make your code easier to read